src/pcap/test/integration/helpers.go

```diff
@@ -176,17 +176,17 @@ func generateCerts(commonName string, dir string) (string, string, string, error
 	certPath := path.Join(dir, "cert.pem")
 	keyPath := path.Join(dir, "private.key")
 	caPath := path.Join(dir, "ca.pem")
-	err = os.WriteFile(certPath, certPEM.Bytes(), os.ModePerm)
+	err = os.WriteFile(certPath, certPEM.Bytes(), 0600)
 	if err != nil {
```

Comment:
I would prefer defining a variable with a name that describes the permissions, then use that. A 'magic number' linter should have noted this.